web/src/user_profile.js

```diff
@@ -235,6 +235,8 @@ function render_user_stream_list(streams, user) {
         },
         $simplebar_container: $("#user-profile-modal .modal__body"),
     });
+    $(".ind-tab").attr("tabindex", "-1");
```

Comment:
So this is bad in a couple ways. One is that it applies to all `ind-tab` elements in the app - not just the ones in this modal.

Second is that it makes it impossible to shift-tab back to these elements.